amp/TEMPLATE/ampTemplate/gisModule/dev/app/js/amp/data/collections/activity-collection.js

```diff
@@ -243,7 +243,7 @@ module.exports = Backbone.Collection
 			var activities = new TempActivityCollection([], {
 				appData : this.appData
 			});
-			activities.url = '/rest/gis/activities/' + aryOfIDs.join(',');
+			activities.url = '/rest/gis/activities?' + aryOfIDs.map(function(id) { return "id="+id; }).join('&');
```

Comment:
here too there is a missing ?

Reply:
Actually here question mark is present. When I tested apparently the other code wasn't triggered.

Reply:
yeah you are right, I must have been looking the previous one